src/tags.rs

```diff
@@ -126,6 +126,13 @@ impl Tags {
                 insert_if_not_empty("ua.os.family", metrics_os, &mut tags);
                 insert_if_not_empty("ua.os.ver", &ua_result.os_version.to_owned(), &mut tags);
                 insert_if_not_empty("ua.browser.ver", ua_result.version, &mut tags);
+                insert_if_not_empty(
```

Comment:
We also want this new tag emitted by the periodic reporter metrics, unfortunately it does not get any tags via `Tags::from_head` (since it's outside of a web request). Like in syncstorage, we probably just need a one off call to gethostname in that code path.

It's too bad we call `gethostname` every request here but I guess it's cheap and currently a bit difficult to do once up front.